src/Illuminate/Support/Number.php

```diff
@@ -2,12 +2,27 @@
 
 namespace Illuminate\Support;
 
+use NumberFormatter;
 use Illuminate\Support\Traits\Macroable;
 
 class Number
 {
     use Macroable;
 
+    /**
+     * Format the number to a fluent human-readable string.
+     *
+     * @param float|int $number
+     * @param string $locale
+     * @return false|string
+     */
+    public static function toHuman($number, $locale = 'en')
+    {
+        $formatter = new NumberFormatter($locale, NumberFormatter::SPELLOUT);
+
+        return $formatter->format($number);
+    }
+
```

Comment:
This is unfortunately dependent on the `ext-intl`, as it wraps the [NumberFormatter](https://www.php.net/manual/en/class.numberformatter.php) class. Assuming this is acceptable, we may want to add a check for the extension in the `NumberFormatter` class, and throw an exception if it is not available. But I understand if this is not worth it. However, given how much code a DIY solution would require, I personally think it is worth it.

Reply:
If this is acceptable, I'll go right ahead and add some more wrappers as the `NumberFormatter` class has a lot of useful helpers.

Reply:
@taylorotwell Hey! Hope it's okay to ping you. Just wanted your opinion regarding wrapping the `NumberFormatter` class, and how we should handle that.